Automattic/jetpack
# Contact Form block: Respect the height of the Spacer block nested between two form fields

#### Changes proposed in this Pull Request:
This PR addresses the issue (https://github.com/Automattic/jetpack/issues/16082) where the set height of the Spacer block inside Contact Form block is not respected.

#### Does this pull request change what data or activity we track or use?
No, it does not.

#### Testing instructions:
1. Create a new page / post
2. Add a Contact Form (Form) block
3. Insert a Spacer block in between two form fields
4. Save the page / post and review the front-end. The height of the Spacer block should be respected

Before:
![Markup on 2021-09-09 at 13:42:26](https://user-images.githubusercontent.com/25105483/132679665-37a76c24-1004-4b26-bf22-6ec5e325582f.png)

After:
![Markup on 2021-09-09 at 13:41:31](https://user-images.githubusercontent.com/25105483/132679696-2261704a-0e8a-42e5-a1ed-a239ac119e92.png)

Fixes https://github.com/Automattic/jetpack/issues/16082
